examples/flutter_gallery/lib/demo/cupertino/cupertino_picker_demo.dart

```diff
@@ -151,9 +92,41 @@ class _CupertinoPickerDemoState extends State<CupertinoPickerDemo> {
     );
   }
 
+  Widget _buildCountdownTimerPicker(BuildContext context) {
+    return new GestureDetector(
+      onTap: () async {
+        await showModalBottomSheet<void>(
```

Comment:
while you're here, can you change this and the one below to use showCupertinoModalPopup?

Reply:
yup!